apollo-federation/src/link/join_spec_definition.rs

```diff
@@ -256,6 +256,19 @@ impl JoinSpecDefinition {
         }
     }
 
+    pub(crate) fn join_type_definition<'schema>(
```

Comment:
We usually use `XXX_directive_definition` or `XXX_type_definition`. In this case, shouldn't it be `join_type_directive_definition`?